diff --git a/node_modules/postcss-discard-duplicates/src/index.js b/node_modules/postcss-discard-duplicates/src/index.js
index 93e4429..7058149 100644
--- a/node_modules/postcss-discard-duplicates/src/index.js
+++ b/node_modules/postcss-discard-duplicates/src/index.js
@@ -54,7 +54,7 @@ function equals(a, b) {
   }
 
   if (a.nodes) {
-    if (a.nodes.length !== b.nodes.length) {
+    if (a.nodes?.length !== b.nodes?.length) {
       return false;
     }
 
